pupil-spa/src/app/virtual-keyboard/virtual-keyboard.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="question-container">
+  <form id='js-question-form' action="/check/question/{{questionNum}}" method="POST" autocomplete="off">
+    <input type="hidden" id="js-form-answer" name="answer" value="">
```

Comment:
@jon-shipley The scope of this ticket doesn't cover form processing, I left these hidden fields (together with the form action) just because we are migrating what we had before. I've removed them as per your suggestion.